src/tscore/ink_memory.cc

```diff
@@ -277,3 +277,15 @@ _xstrdup(const char *str, int length, const char * /* path ATS_UNUSED */)
   }
   return nullptr;
 }
+
+ats_scoped_str &
+ats_scoped_str::operator=(std::string_view s)
+{
+  this->clear();
```

Comment:
So this is fixing a leak of _r from the original version?

Reply:
I think so.